### DISCUSSION TITLE
Option to quickly capture fullscreen screenshot&auto copy it to clipboard

### DISCUSSION THREAD

**Discussion by SPLESHER1** [flameshot-org/flameshot#2648] [Q&A] [answered]
Hey,
I'd like to request feature,
1.Option to automatically copy selection or fullscreen screenshots to clipboard
2.Shortcut key for capturing fullscreen screenshot

**Comment by mmahmoudian** [comment, marked as answer]
Already exists. If you are on Linux or macOS you can use `flameshot full --clipboard`. You can read more about the CLI (CommandLine Interface) in the following link:

https://flameshot.org/docs/advanced/commandline-options/

For the time being we do not support Windows CLI for Windows, and you can follow that feature request in #2118.

**Comment by SPLESHER1** [comment]
Im using Windows 11 Pro unfortunately, I hope you will add support for it soon
